src/Minsk/CodeAnalysis/Emit/Emitter.cs

```diff
@@ -57,7 +57,7 @@ private Emitter(string moduleName, string[] references)
                 }
             }
 
-            var builtInTypes = new List<(TypeSymbol type, string MetadataName)>()
+            var builtInTypes = new List<(TypeSymbol type, string metadataName)>()
```

Comment:
If you want to change this, it should be the other way around. As you can see in this [comment](https://github.com/dotnet/runtime/issues/27939#issuecomment-531420515), the .NET Runtime team decided to use the Pascal naming convention (also the summary was written by the Minsk master @terrajobst, so another reason why use this convention  :) )

Reply:
That's the convention for the BCL, which, let's be honest, is an obvious mistake. 😉 
We don't need to repeat that mistake here.

In all seriousness, most of the tuples you'll see in the wild seem to use camelCase. The BCL convention is not really followed that much.

Reply:
@ostorc @ltrzesniewski 
There's another place in the codebase where pascal case is used - https://github.com/terrajobst/minsk/blob/7223f8249c0693ab1cbe080b66681a5178c96467/src/Minsk/CodeAnalysis/Emit/Emitter.cs#L38
(I discovered it after opening this PR)

Should we ask Immo for the executive decision? @terrajobst what do you say?

Reply:
@ltrzesniewski for me it feels quite unnatural to access tuple member which was declared with camel case. It just doesn't feel right. If I want to be it in camel case I deconstruct the tuple into local variables.

Reply:
But again, this discussion was already held in the runtime issue. We should probably just await the response from @terrajobst

Reply:
We decided to go with PascalCase for one reason: usage. When accessing tuple members, they should behave like any other public field/property, which is PascalCase. This also makes it easier to refactor the code base from tuples to custom types. So yeah, PascalCasing is the correct casing for tuples. I consider any deviation from this in Minks to be a bug :-)

Reply:
Pushed a new commit reflecting the final decision.